website/ref/index.toml

```diff
@@ -1,55 +1,47 @@
 [[articles]]
 name = "language"
-title = "The Language"
+title = "The Language Specification"
 
 [[articles]]
 name = "builtin"
-title = "The Builtin Module"
+title = "Builtin Functions and Variables (builtin:)"
```

Comment:
`builtin:` is not the canonical way to use the builtin module, only for introspection. So I'd rather this be left out of the title. A paragraph inside `builtin.html` can explain the fact that `$builtin:` can be used for introspection.

Reply:
Done